src/compiler/scala/tools/nsc/fsc/CompileSocket.scala

```diff
@@ -139,7 +139,7 @@ class CompileSocket extends CompileOutputCommon {
     val file = portFile(port)
     // 128 bits of delicious randomness, suitable for printing with println over a socket,
     // and storage in a file -- see getPassword
-    val secretDigits = new BigInteger(128, new SecureRandom()).toString.getBytes("UTF-8")
+    val secretDigits = new BigInteger(128, new SecureRandom()).toString.getBytes(java.nio.charset.StandardCharsets.UTF_8)
```

Comment:
You imported below but not here. My preference would be the compromise `StandardCharsets.UTF_8`

Reply:
When I grepped at 2 a.m., I cut/pasted from JavaMirrors.

Reply:
I went scorched-earth because no one looks at `UTF_8` and goes what is that? "If it compiles, it styles."